app/(auth)/password-reset/[token]/page.tsx

```diff
@@ -14,13 +14,13 @@ const PasswordReset = () => {
 
     const { resetPassword } = useAuth({ middleware: 'guest' })
 
-    const [email, setEmail] = useState('')
-    const [password, setPassword] = useState('')
-    const [passwordConfirmation, setPasswordConfirmation] = useState('')
-    const [errors, setErrors] = useState([])
-    const [status, setStatus] = useState(null)
+    const [email, setEmail] = useState<string>('')
```

Comment:
it's not needed, because ts detects the type by the given value

Reply:
done, thanks